internal/scraper/scraper_test.go

```diff
@@ -169,18 +194,20 @@ func TestValidateMetrics(t *testing.T) {
 //
 // Optionally, this function will update the golden files if the
 // -update-golden flag was passed to the test.
-func verifyProberMetrics(t *testing.T, name string, prober prober.ProbeFn, setup func(t *testing.T) (string, func()), config bbeconfig.Module, basicMetricsOnly bool) {
+func verifyProberMetrics(t *testing.T, name string, prober ProbeFn, setup func(t *testing.T) (string, func()), config ConfigModule, basicMetricsOnly bool) {
 	ctx, cancel := context.WithTimeout(context.Background(), 10*time.Second)
 	defer cancel()
 
 	summaries := make(map[uint64]prometheus.Summary)
 	histograms := make(map[uint64]prometheus.Histogram)
-	logger := &testLogger{w: ioutil.Discard}
+	// logger := &testLogger{w: ioutil.Discard}
+
+	logger := kitlog.NewJSONLogger(kitlog.NewSyncWriter(os.Stdout))
```

Comment:
Can you explain what's happening here 🙏

Reply:
Yes! I was having trouble with the tests so I wanted to see the log output (the test logger is just discarding them, I think). I forgot to revert the change.